### ISSUE TITLE
Call without arguments

### ISSUE THREAD

**Issue by Eitot** [scour-project/scour#34]
When I just type ``scour``, the console will show the version number and copyright notice, but it won’t terminate the Python process and return to the command line automatically. Is this intentional? Normally, I would expect that a call without any arguments either returns nothing or ``--help``.

Pressing control + c will also give me this output:
```
File "/usr/local/Cellar/scour/0.32/libexec/bin/scour", line 8, in <module> load_entry_point('scour==0.32', 'console_scripts', 'scour')()
File "/usr/local/Cellar/scour/0.32/libexec/lib/python2.7/site-packages/scour/scour.py", line 3321, in run
  start(options, input, output)
File "/usr/local/Cellar/scour/0.32/libexec/lib/python2.7/site-packages/scour/scour.py", line 3295, in start
  in_string = input.read()
KeyboardInterrupt
```

Note, I also have this when I install Scour with pip.

**Comment by Ede123** [comment]
If you do not provide an input file using the `-i` command line switch Scour expects the input to come from `stdin`, e.g. by using `scour < input.svg`. That's why Scour doesn't terminate in this case you describe.

I'm not sure if we can catch this without breaking the (intended) functionality. Maybe somebody is more experienced working with `stdin` in Python?

**Comment by Eitot** [comment]
But is this method working for you? Scour always expects two arguments.

**Comment by Ede123** [comment]
Yes, works perfectly on Windows 7 with Python 2.7 and 3.4 (and I think I tested on Ubuntu with Python 2.7, too)...

How do you figure Scour expects two arguments? The only relevant check is [here](https://github.com/codedread/scour/blob/master/scour/scour.py#L3257) and if `infilename`/`outfilename` is unset `stdin`/`stdout` are sued instead.

Maybe you can provide a command line that is failing for you, then I can check on my machine this evening.

**Comment by Ede123** [comment]
Call from commandline without arguments fixed in 4f23ea7a342ef9e191a27d99db7cbb1063d975a3

@Eitot - If usage of `stdin/stdout` as input/output still does not work for you feel free to open a new issue with some more details.
